infra/chromadb_compaction/lambdas/compaction/metadata_handler.py

```
@@ -0,0 +1,328 @@
+                try:
+                    logger.info(
+                        "Getting ChromaDB collection", collection=database
+                    )
+                    collection_obj = chroma_client.get_collection(database)
+
+                    logger.info(
+                        "Successfully got ChromaDB collection",
+                        collection=database,
+                    )
+                except (
+                    Exception
```

Ruff (0.14.1)

Do not catch blind exception: `Exception`

(BLE001)